Modules/_collectionsmodule.c

```diff
@@ -1050,8 +1050,10 @@ deque_index(dequeobject *deque, PyObject *const *args, Py_ssize_t nargs)
         start = stop;
     assert(0 <= start && start <= stop && stop <= Py_SIZE(deque));
 
-    /* XXX Replace this loop with faster code from deque_item() */
-    for (i=0 ; i<start ; i++) {
+    for (i=0 ; i < start - BLOCKLEN ; i += BLOCKLEN) {
```

Comment:
PEP 7 requires spaces around `=` and no spaces before `;`.

Reply:
Please use your talents to focus on something relevant.  Over-aggressive interpretations of PEP 7 are waste of time.